ntoskrnl/ps/query.c

```diff
@@ -1172,7 +1172,7 @@ NtSetInformationProcess(IN HANDLE ProcessHandle,
         case ProcessWx86Information:
 
             /* Check buffer length */
-            if (ProcessInformationLength != sizeof(HANDLE))
+            if (ProcessInformationLength != sizeof(ULONG))
```

Comment:
Are you sure you change the correct case? This is `ProcessWx86Information`, not `ProcessWow64Information`

Reply:
EDIT: indeed you're correct here, for `ProcessWx86Information` this is indeed a ULONG.